packages/media-query/src/use-media-query.ts

```diff
@@ -37,17 +40,18 @@ export function useMediaQuery(
     }))
   })
 
-  useSafeLayoutEffect(() => {
+  useEffect(() => {
     // set initial matches
-    if (ssr) {
-      setValue(
-        queries.map((query) => ({
-          media: query,
-          matches: env.window.matchMedia(query).matches,
-        })),
-      )
-    }
+    setValue(
+      queries.map((query) => ({
+        media: query,
+        matches: env.window.matchMedia(query).matches,
+      })),
+    )
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
 
+  useSafeLayoutEffect(() => {
```

Comment:
I think it's okay to move this code into the `useEffect` for simplicity.

Reply:
refactored be45b1a5100079ed78451652a0f038f716a82680